plugins/content/config/model.schema

```diff
@@ -51,17 +51,15 @@
           "isSetting": false,
           "inputType": "Checkbox",
           "validators": [],
-          "title": "Enable accessibility?",
-          "help": "If enabled, accessibility features will be enabled in this course"
+          "title": "Enabled?"
         },
         "_isEnabledOnTouchDevices": {
           "type": "boolean",
           "default": false,
           "isSetting": false,
           "inputType": "Checkbox",
           "validators": [],
-          "title": "Enable accessibility on touch devices?",
-          "help": "If enabled, accessibility features will be enabled on touch devices - useful if you need accessibility on mobile browsers."
```

Comment:
Is there any reason to remove the help?

Reply:
Again, it's not adding anything that you wouldn't already have gleaned from reading the title!

Reply:
If feel it's equivalent to [doing this](https://github.com/adaptlearning/documentation/blob/master/01_cross_workstream/style_guide.md#comments)